docker-app/qfieldcloud/core/drf_utils.py

```diff
@@ -74,8 +76,8 @@ def _parse_definition(self, definition: str) -> tuple[str, dict[str, str]]:
     def remove_invalid_fields(
         self,
         queryset: QuerySet,
-        fields: list[str],
-        view: viewsets.ModelViewSet,
+        fields: Iterable[str],
+        view: views.APIView,
         request: HttpRequest,
```

Comment:
Are we sure this is actually a `HttpRequest` and not `drf.Request`? I have a guess it will most probably be a wrong type here with the `HttpRequest`.

Reply:
The class `QfcOrderingFilter` inherits from `filters.OrderingFilter`, with the later to be part of Django REST Framework. Taking into account that the `OrderingFilter` works with DRF's Request, the most likely is the `request` a DRF one.

Reply:
Changed to drf.request